.github/workflows/prebuild.yml

```diff
@@ -2,9 +2,9 @@ name: Pre-build
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, 'release/**']
   pull_request:
-    branches: [ main ]
+    branches: [ main, 'release/**']
```

Comment:
Small nit, can we be consistent with the quotes?

Reply:
Sure, done